lib/grammers-client/examples/downloader.rs

```diff
@@ -0,0 +1,195 @@
+//! Example to download all messages and media from a chat.
+//!
+//! The `TG_ID` and `TG_HASH` environment variables must be set (learn how to do it for
+//! [Windows](https://ss64.com/nt/set.html) or [Linux](https://ss64.com/bash/export.html))
+//! to Telegram's API ID and API hash respectively.
+//!
+//! Then, run it as:
+//!
+//! ```sh
+//! cargo run --example downloader -- CHAT_NAME
+//! ```
+//!
+//! Messages will be printed to stdout, and media will be saved in the ```target/``` folder locally, named
+//! message-[id].[jpg/mp4]
+//!
+extern crate core;
+extern crate grammers_client;
+extern crate grammers_session;
+extern crate grammers_tl_types;
```

Comment:
Is there any reason to do this? None of the other crate examples do.

Reply:
no reason. It is a leftover